src/Codec/Xlsx/Types/Internal/CommentTable.hs

```diff
@@ -75,6 +75,10 @@ parseComment authors cur = do
     authorId <- cur $| attribute "authorId" >=> decimal
     let author = fromJustNote "authorId" $ M.lookup authorId authors
     return (ref, Comment txt author True)
+    -- ^ Why is this always True? In testXlsx, we specify some comments as visible
+    -- (e.g., bob) and some as invisible (e.g., alice). I don't understand how these
+    -- test cases (particularly 'toXlsxEither: properly formatted') were passing
+    -- before the refactor.
```

Comment:
There are 3 test cases that I wasn't able to resolve: `toXlsxEither: properly formatted`, `write . read == id` and `write . fast-read == id`. They all fail based on some data in testXlsx, but I'm confused as to why the test case didn't fail before these changes. As part of the conversions carried out in these tests we expect to get `comment2 = Comment (XlsxRichText [rich1, rich2]) "alice" False` (line 194 in TestXlsx.hs). This comment should be invisible, but in the code above we are always defining comments as visible?

Reply:
Fixing this in another PR

Reply:
Other PR is here, I'd expect this to resolve the test case failures here: https://github.com/qrilka/xlsx/pull/163